packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -26,12 +29,14 @@ function getBaseMafsGraphProps(): MafsGraphProps {
         graph: {type: "segment"},
         containerSizeClass: "small",
         onChange: () => {},
+        labels: [],
     };
 }
 
 describe("StatefulMafsGraph", () => {
     let userEvent: UserEvent;
     beforeEach(() => {
+        setDependencies(testDependencies);
```

Comment:
Was getting an error saying dependencies were not set for Perseus here, so followed the instructions in the error to get this set up. Looked up each part to set up the imports.